app/models/access_limit.rb

```diff
@@ -13,12 +12,6 @@ def user_uids_are_strings
     end
   end
 
-  def auth_bypass_ids_are_uuids
-    unless auth_bypass_ids.all? { |id| UuidValidator.valid?(id) }
-      errors.add(:auth_bypass_ids, ["contains invalid UUIDs"])
-    end
-  end
```

Comment:
~Should this validation still happen, just on the auth_bypass_ids column on editions in the previous commit?~ Ah, it happens in edition.rb. Glad you removed the duplicate code!